tutorials/vue/img.md

```diff
@@ -0,0 +1,139 @@
+To use the `img` tag in vue, you must use the `v-bind:src` directive or `:src` for short.
+Be sure to enclose the image address in single quotes on top of the double quotes.
+
+```javascript
+  const app = new Vue({
+    template: `
+      <div style="width: 50%">
+       <img :src="'../../assets/logo.png'" />
+      </div>
+    `,
+  });
+  app.$mount("#content");
+```
+
+<div id="content"></div>
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12"></script>
```

Comment:
Use unpkg instead like in our [Vue CDN](https://masteringjs.io/tutorials/vue/cdn) tutorial